.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
Ah, you're right, there is no recent Sherpa conda build in the `sherpa` or `cxc` channel.
I'll ask about this in the Sherpa tracker.

Building Sherpa from source is a possibility and would be very useful to test for us.
I've never tried pip install, but `python setup.py install` basically works.
@bsipocz - Up to you whether you want to try adding this in this PR or leave it for the future.